src/test/java/org/jpeek/metrics/cohesion/CAMCTest.java

```diff
@@ -39,6 +39,8 @@
  * @since 0.1
  * @checkstyle AbbreviationAsWordInNameCheck (5 lines)
  * @checkstyle JavadocMethodCheck (500 lines)
+ * @checkstyle EmptyLinesCheck (500 lines)
+ * @checkstyle LineLengthCheck (500 lines)
```

Comment:
@sergey-karazhenets please, get rid of them both. It's a bad practice to have empty lines in methods: http://www.yegor256.com/2014/11/03/empty-line-code-smell.html Also, when you _need_ to have a long line, put the Checkstyle suppression right in front of it